drivers/audio/cxd56_src.h

```diff
@@ -0,0 +1,127 @@
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name NuttX nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+#ifndef __DRIVERS_AUDIO_CXD56_SRC_H
+#define __DRIVERS_AUDIO_CXD56_SRC_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <pthread.h>
+#include <mqueue.h>
+
+#include <nuttx/audio/audio.h>
+#include <nuttx/config.h>
+
+#ifdef CONFIG_AUDIO
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#ifndef CONFIG_SMP_NCPUS
+#  define CONFIG_SMP_NCPUS  4
+#endif
+
+#ifndef CONFIG_CXD56_AUDIO_SRC_CPU
+#  define CONFIG_CXD56_AUDIO_SRC_CPU  2
+#endif
+
+#ifndef CONFIG_CXD56_AUDIO_SRC_STACKSIZE
+#  define CONFIG_CXD56_AUDIO_SRC_STACKSIZE  768
+#endif
+
+#ifndef CONFIG_CXD56_SRC_MSG_PRIO
+#  define CONFIG_CXD56_SRC_MSG_PRIO  1
+#endif
+
+#ifndef SRC_SINC_BEST_QUALITY
+#  define SRC_SINC_BEST_QUALITY     0
+#endif
+
+#ifndef SRC_SINC_MEDIUM_QUALITY
+#  define SRC_SINC_MEDIUM_QUALITY   1
+#endif
+
+#ifndef SRC_SINC_FASTEST
+#  define SRC_SINC_FASTEST          2
+#endif
+
+#ifndef SRC_ZERO_ORDER_HOLD
+#  define SRC_ZERO_ORDER_HOLD       3
+#endif
+
+#ifndef SRC_LINEAR
+#  define SRC_LINEAR                4
+#endif
+
+#define AUDIO_APB_SRC_FINAL  (1 << 4) /* Last buffer in SRC processing */
+
+/****************************************************************************
+ * Public Types
+ ****************************************************************************/
+
+typedef struct SRC_STATE_TAG SRC_STATE;
+
+typedef struct
+{       const float *data_in;
```

Comment:
@jerpelea 
Though I'm not sure why nxstyle does not report a warning, but ```const float ..``` should be placed in the next line. Also, indentation spaces should be 2.